lstchain/io/tests/test_eventselection.py

```diff
@@ -44,16 +44,31 @@ def test_dl3_fixed_cuts():
     temp_cuts.fixed_gh_cut = 0.7
     temp_cuts.fixed_theta_cut = 0.2
     temp_cuts.allowed_tels = [1, 2]
+    temp_cuts.fixed_gh_max_efficiency = 0.8
```

Comment:
Again giving both the fixed config and the energy dependent config. These should be clearly separated.

Reply:
Will do so.